src/System.Net.Security/src/System/Net/SecureProtocols/NegoState.Unix.cs

```diff
@@ -19,140 +19,135 @@ namespace System.Net.Security
     //
     // This is part of the NegotiateStream PAL.
     //
-    internal class NegoState
+    internal partial class NegoState
     {
-        internal const int MaxReadFrameSize = 64 * 1024;
-        internal const int MaxWriteDataSize = 63 * 1024; // 1k for the framing and trailer that is always less as per SSPI.
-
-        internal NegoState(Stream innerStream, bool leaveStreamOpen)
+        internal IIdentity GetIdentity()
         {
             throw new PlatformNotSupportedException();
         }
 
-        internal static string DefaultPackage
+        internal static string QueryContextAssociatedName(SafeDeleteContext securityContext)
         {
-            get
-            {
-                throw new PlatformNotSupportedException();
-            }
+            throw new PlatformNotSupportedException();
```

Comment:
Ok, thanks, @shrutigarg.  @vijaykota, this is an example of how separate PRs are difficult.  I reviewed and commented on code which was apparently already out of date, and it doesn't make sense for Shruti to fix in this PR because the code is already deleted from another, so now this old code is going to get merged.